humble/distribution.yaml

```diff
@@ -5444,6 +5444,20 @@ repositories:
       url: https://github.com/ros-visualization/python_qt_binding.git
       version: humble
     status: maintained
+  qbshin-ros:
+    release:
+      packages:
+      - qb_softhand_industry
+      - qb_softhand_industry_description
+      - qb_softhand_industry_driver
+      - qb_softhand_industry_msgs
+      - qb_softhand_industry_ros2_control
+      - qb_softhand_industry_srvs
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://GiulianoDami@bitbucket.org/qbrobotics/qbshin-ros2-release.git
```

Comment:
I don't believe including your username in the URL here is going to be valid.

Reply:
You are right, I'll fix this.